<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>res</class>
 <widget class="QWidget" name="res">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit {
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,stop:0 #003366, stop:1 #000033);
color: #FFFFFF;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查看折线图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
  font-size: 30px; // 设置字体大小
   text-align: center;  // 设置文本居中对齐
   background: qlineargradient(
     spread:pad, x1:0, y1:0, x2:0, y2:1,
     stop:0 #003366, stop:1 #000033
   );
   color: #FFFFFF;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
